### ISSUE TITLE
Question rather than issue

### ISSUE THREAD

**Issue by ghost** [DKingCN/nethunter_kernel_huawei_angler#1]
Is included kernel for Oreo 8.0 or 8.1, are the patches included in provided boot image  ? Thank you for your effort.

**Comment by DKingAlpha** [comment]
This is for Oreo release 6, one of a release of Android 8.0 which might be a bit out of date.
And yes the patches are included in that `Image.gz-dtb`. It is provided as a final release.

However, this patch&config does not work well. It will freeze while trying to switch wireless adapter to monitor mode. I failed to fix it.

**Comment by ghost** [comment]
Yeah, I faced the same problem even on nethunter , nighty kernel. Any chance of creating an up to date 8.1 one image.gz-dtb with at least rubber ducky support ?

Thanks for all your effort.

One the  second thought I am up for a challenge if you are keen to help. Have you tried modify elementalx kernel for lineage os to support most of the nethunter features ?

**Comment by DKingAlpha** [comment]
HID attack works in that image. I patched them. I think this is your rubber ducky attack?

Anyway I have left this device (turning to Oneplus 5T)

This kernel is compiled from AOSP source. Can't be better if based on elem-x or whatever.
